liam2/utils.py

```diff
@@ -1081,6 +1082,8 @@ def validate_value(v, target, context):
         validate_dict(v, target, context)
     elif isinstance(target, list):
         validate_list(v, target, context)
+    elif isinstance(target, numbers.Number) or isinstance(target, bool):

```

Comment:
this shouldn't be needed since you are using None as validator for the 'value' keyword